packages/subgraph/commons/ids.ts

```diff
@@ -58,15 +32,3 @@ export function getPluginInstallationId(
   }
   return null;
```

Comment:
Is this intentional?

Reply:
yes,
`  if (installationIdTupleEncoded) {
  =>  return Bytes.fromHexString(
      crypto
        .keccak256(
          ByteArray.fromHexString(installationIdTupleEncoded.toHexString())
        )
        .toHexString()
    );
  }
  return null;`

Reply:
NVM, my github was buggy/showed and old commit.